symplectic_manifolds.tex

```diff
@@ -1,35 +1,33 @@
+\documentclass[main.tex]{subfiles}
+\begin{document}
+
 \chapter{Symplectic geometry}
-\section{Symplectic vector spaces}
+Symplectic and Poisson structures came from the structures observed in analytical mechanics (originated from the need of studying complex dynamical systems in exact fashion, like motions of planets, oscillations of many particles systems, etc.).
+\section{Symplectic forms on vectors spaces}
+\subsection{Skew-symmetric bilinear maps}
 \begin{definition}
-	A \textbf{symplectic form} is a non-degenerate, skew-symmetric bilinear form. A vector space $V$ equipped with a symplectic form on $V$ is called a \textbf{symplectic vector field}.
+	Let $V$ be an $\R$-vector space with $\dim V=m$. Let $\Omega:V\times V\to\R$ be a bilinear map. $\Omega$ is \textbf{skew-symmetric} iff $\Omega(v,w)=-\Omega(w,v)$ $\forall v,w\in V$.
```

Comment:
Never start a sentence with a symbol. `Ω is skew-symmetric` --> `We say Ω is skew-symmetric`.
Put the definition of skew-symmetry in a display environment:
```
$$
    \Omega(v,w)=-\Omega(w,v), \qquad \forall v,w \in V
$$
```